Tech-Talk-LA/TechTalk
# Basic Authentication -- can add user's device to db

## Description ## 
I implemented the ability to add a device to the database. This would happen when a user for signs up for an account. I set up the outline of an Express server so I can test this functionality. 

## What I changed ##
Not much. I added a 'Devices' table to the db, which will have an authToken attached to it, as well as a created_at field. Eventually, the server will check that the authToken that gets sent corresponds to the deviceID in the db, and that the authToken has not expired. 

## How to test ##
npm run server (starts the server(
On postman, make a post request to localhost:3000/auth 
You should back an object with a deviceid, and authToken, and a last_updated filed
